db/migrate/20230503003831_add_user_to_like.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
* [ ]  Same here, Kindly edit this reference so the foreign key supposes to be `author_id` but what you did here will make Active Record create it as `user_id`